### PR TITLE
fix cpu moe

### PR DESCRIPTION
**Pull request by pathfinder-pf**
## Accuracy Tests
```
JAX_PLATFORMS=cpu JAX_COMPILATION_CACHE_DIR=/tmp/jit_cache python3 -u -m sgl_jax.launch_server --model-path Qwen/Qwen3-30B-A3B --trust-remote-code --dist-init-addr=0.0.0.0:10011 --nnodes=1 --tp-size=1 --device=cpu --random-seed=27 --node-rank=0 --mem-fraction-static=0.9 --chunked-prefill-size=8192 --download-dir=/tmp --dtype=float32 --precompile-bs-paddings 1 8 --max-running-requests 64 --max-total-tokens 257536 --skip-server-warmup --attention-backend=native --precompile-token-paddings 128 --page-size=64 --disable-overlap-schedule --log-requests --log-requests-level=3 --disable-precompile --model-layer-nums 1
```
```
curl http://localhost:30000/v1/chat/completions   -H "Content-Type: application/json"   -H "Authorization: Bearer $OPENAI_API_KEY"   -d '{
    "model": "gpt-4o",
    "messages": [
      {
        "role": "user",
        "content": "请用一句话解释什么是量子纠缠。"
      }
    ],
    "max_tokens": 1
  }'
```

## Checklist

- [ ] Please use English, otherwise it will be closed.
- [ ] The purpose of the PR, or link existing issues this PR will resolve.
- [ ] The test plan, such as providing test command.
- [ ] (Optional) The necessary documentation update.

### CODE REVIEW

File: python/sgl_jax/srt/layers/moe.py [hunk 1]
```diff
@@ -366,6 +366,7 @@ def _gmm_compute(self, x, group_sizes, w0_kernel, w1_kernel, wo_kernel, group_of
             preferred_element_type=self.dtype,
             tiling=tiling_gate,
             group_offset=group_offset,
+            interpret=jax.local_devices()[0].platform != "tpu",
```
Comment (jimoosciuc) [inline python/sgl_jax/srt/layers/moe.py]: It looks like we can directly reuse the is_tpu_runtime function? It's located in sgl_jax.srt.utils.jax_utils

Comment (pathfinder-pf) [reply python/sgl_jax/srt/layers/moe.py]: get
